app/src/Commands/Pack/ExportCommand.php

```diff
@@ -44,6 +44,7 @@ public function configure() : void
         $this
             ->addArgument("instance", InputArgument::REQUIRED, "Name of the instance to export")
             ->addOption("global", "g", InputOption::VALUE_NONE, "Determines if the instance is global")
+            ->addOption("cron", "c", InputOption::VALUE_NONE, "Must be use for triggering export by cron")
```

Comment:
Must be **used** for triggering **an** export ...

Reply:
Done!